.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: "CI"
 
 on:
   push:
-    branches: [ "master" ]
+    branches: [ "master", "continue-v0.14.x" ]
   pull_request:
-    branches: [ "master" ]
+    branches: [ "master", "continue-v0.14.x" ]
```

Comment:
Is this necessary?

Reply:
CI wasn't building in this PR and I wanted to be sure that `libtinfo` was verified by CI to not be included in the binary. So, I think so? I can remove it in a later commit once CI builds.

Reply:
I assume `continue-v0.14.x` will only receive backports and isn't going to be merged back into `master`, so it doesn't matter if it's ever removed, right? This is meant to be a short-lived branch, right?

Reply:
Yup! I don't anticipate making a `v0.14.9` release. This is just to fix the `libtinfo` regression in `v0.14.7`